test/e2e/argo-workflows/cluster-agent.yaml

```diff
@@ -182,6 +179,8 @@ spec:
                   value: "true"
                 - name: DD_LEADER_ELECTION
                   value: "true"
+                - name: DD_KUBERNETES_APISERVER_POLL_FREQ
+                  value: "20"
```

Comment:
The only required envvar here is `DATADOG_HOST` because of the datadog agent library right?

Reply:
Oh I understand what your saying. Yes, `DATADOG_HOST` is required because of https://github.com/zorkian/go-datadog-api/blob/master/client.go#L38 to use the fake datadog instead.